src/stores/osimRuntime.ts

```diff
@@ -44,10 +44,17 @@ const OsimRuntime = z.object({
 type OsimRuntime = z.infer<typeof OsimRuntime>;
 
 const runtime = ref<OsimRuntime>({
-  env: '',
-  backends: { osidb: '', osidbAuth: '', bugzilla: '', jira: '', errata: '', jiraDisplay: '' },
-  osimVersion: { rev: '', tag: '', timestamp: '' },
-  error: '',
+  env: 'dev',
+  backends: {
+    osidb: 'http://osidb-service:8000',
+    osidbAuth: 'kerberos',
+    bugzilla: 'http://bugzilla-service:8001',
+    jira: 'http://jira-service:8002',
+    errata: 'http://errata-service:8003',
+    jiraDisplay: 'http://jira-service:8002',
+  },
+  osimVersion: { rev: 'dev', tag: 'dev', timestamp: '1970-01-01T00:00:00Z' },
+  error: 'error',
```

Comment:
why these changes?

Reply:
It wasn't really necessary to add default 'dummy' values to osimRuntime ref, but I considered they are useful as example and that they shouldn't be empty strings under any case. Also considering they are used on:
[Local Configuration](https://github.com/RedHatProductSecurity/osim/blob/main/docs/overview.md?rgh-link-date=2024-09-16T12%3A41%3A58Z#local-configuration)